pkgs/by-name/ne/netmaker/package.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, installShellFiles
+, nix-update-script
+, withServer ? true
+, withCLI ? false
+, isPro ? false
+, ...
+}:
+buildGoModule rec {
+  pname = builtins.concatStringsSep "-" (
```

```suggestion
  pname = lib.concatStringsSep "-" (
```
if there is a lib variant of a builtin that is always prefered.